ksql-parser/src/test/java/io/confluent/ksql/parser/util/TypeUtilTest.java

```diff
@@ -132,11 +131,13 @@ public void shouldGetCorrectStructKsqlType() throws Exception {
                                                                                         .MAP));
     assertThat(((Struct) type7).getItems().get(5).getRight().getKsqlType(), equalTo(Type.KsqlType
                                                                                         .STRUCT));
-
+    Type type8 = TypeUtil.getKsqlType(Schema.INT64_SCHEMA);
```

Comment:
This looks like it should be in a different test.

Reply:
It's a duplicate. It's already in the primitive types test. Removed it.